frost-core/src/frost.rs

```diff
@@ -64,19 +65,60 @@ where
     }
 }
 
-impl<C> From<&SigningPackage<C>> for Rho<C>
+/// A list of binding factors and their associated identifiers.
+#[derive(Clone)]
+pub struct BindingFactorList<C: Ciphersuite>(Vec<(Identifier<C>, Rho<C>)>);
```

Comment:
Should we use a `BTreeMap`?

Reply:
Hm! I added a comment as a TODO so we can test it out later and see how the ergonomics of it and the performance feel